src/Bridges/ApplicationLatte/UIMacros.php

```diff
@@ -76,7 +76,7 @@ public function macroControl(MacroNode $node, PhpWriter $writer)
 		return ($name[0] === '$' ? "if (is_object($name)) \$_l->tmp = $name; else " : '')
 			. '$_l->tmp = $_control->getComponent(' . $name . '); '
 			. 'if ($_l->tmp instanceof Nette\Application\UI\IRenderable) $_l->tmp->redrawControl(NULL, FALSE); '
-			. ($node->modifiers === '' ? "\$_l->tmp->$method($param)" : $writer->write("ob_start(); \$_l->tmp->$method($param); echo %modify(ob_get_clean())"));
+			. ($node->modifiers === '' ? "\$_l->tmp->$method($param)" : $writer->write("ob_start(); echo \$_l->tmp->$method($param); echo %modify(ob_get_clean())"));
```

Comment:
It was added by commit: Implemented possibility of helpers for "control" macro https://github.com/nette/nette/commit/aa9ee5bcf28cced776485df10a5a58fa31508257